python/cudf_polars/cudf_polars/utils/config.py

```diff
@@ -149,9 +187,9 @@ class StreamingExecutor:
     fallback_mode: StreamingFallbackMode = StreamingFallbackMode.WARN
     max_rows_per_partition: int = 1_000_000
     cardinality_factor: dict[str, float] = dataclasses.field(default_factory=dict)
-    parquet_blocksize: int = 1_000_000_000
+    parquet_blocksize: int = 0
     groupby_n_ary: int = 32
-    broadcast_join_limit: int = 4
+    broadcast_join_limit: int = 0
     shuffle_method: ShuffleMethod | None = None
```

Comment:
Is there a reason to use `0` instead of `None`? They probably semantically mean the same thing in this context, just from a Python perspective I think `None` is more common.

Reply:
If we do `broadcast_join_limit: int | None = None` we need to add `assert isinstance(broadcast_join_limit, int)` when this config value is actually invoked in other places (otherwise mypy complains). Therefore, this entirely because those assertions annoy me :)